Modules/Devices/src/Devices.Infrastructure/Persistence/Database/EntityConfigurations/PnsRegistrationEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Devices.Infrastructure.Persistence.Database.EntityConfigurations;
+public class PnsRegistrationEntityTypeConfiguration : IEntityTypeConfiguration<PnsRegistration>
+{
+    public void Configure(EntityTypeBuilder<PnsRegistration> builder)
+    {
+        builder.ToTable("PnsRegistration");
+
+        builder.HasKey(x => x.Handle);
```

Comment:
Why would the handle be the primary key of this table?

Reply:
I had decided on using the Handle as the key because it would be the most "unique" combination of characters, but in hindsight it really doesn't make much sense as we can update the handle value. I believe the DeviceId would work better here then, as there can only be one registration per device, correct?

Reply:
Exactly 👍